gallery/src/components/NxTransferList/NxTransferListPage.tsx

```diff
@@ -59,13 +61,31 @@ const NxTransferListPage = () =>
             </NxTable.Row>
             <NxTable.Row>
               <NxTable.Cell><NxCode>selectedItems</NxCode></NxTable.Cell>
-              <NxTable.Cell>Set</NxTable.Cell>
+              <NxTable.Cell>Set | Array</NxTable.Cell>
               <NxTable.Cell>Yes</NxTable.Cell>
               <NxTable.Cell></NxTable.Cell>
               <NxTable.Cell>
-                A Set containing the ids of the items which are selected – that is, the ones which should appear
-                on the right side of the component. Be sure to always provide a fresh <NxCode>Set</NxCode> instance
-                when updating this prop, rather than mutating the same <NxCode>Set</NxCode>.
+                A Set, or an Array if <NxCode>allowReordering</NxCode> is set to true, containing the ids of the
+                items which are selected – that is, the ones which should
+                appear on the right side of the component. Be sure to always provide a fresh
+                <NxCode>Set</NxCode> or <NxCode>Array</NxCode> instance
+                when updating this prop, rather than mutating the same <NxCode>Set</NxCode> or <NxCode>Array</NxCode>.
+              </NxTable.Cell>
+            </NxTable.Row>
+            <NxTable.Row>
+              <NxTable.Cell><NxCode>allowReordering</NxCode></NxTable.Cell>
+              <NxTable.Cell>boolean</NxTable.Cell>
+              <NxTable.Cell>No</NxTable.Cell>
+              <NxTable.Cell>false</NxTable.Cell>
+              <NxTable.Cell>
```

Comment:
This description makes no mention of the buttons allowing the _user_ to reorder the items

Reply:
> This description makes no mention of the buttons allowing the _user_ to reorder the items

Yup, updated the description here: d098925532af2c05b55be1e7b1bd141f26e0352a